packages/jest-validate/src/__tests__/validate.test.js

```diff
@@ -75,20 +75,6 @@ test('omits null and undefined config values', () => {
   });
 });
 
-test('recursively omits null and undefined config values', () => {
```

Comment:
This should now assert on different items. I'll try to send an update to this branch later, but if somebody is faster, that's great

Reply:
Was this addressed?

Reply:
@thymikee ping 🙂

Reply:
updated